rpc/handler.go

```diff
@@ -220,12 +218,12 @@ func (h *handler) cancelServerSubscriptions(err error) {
 // startCallProc runs fn in a new goroutine and starts tracking it in the h.calls wait group.
 func (h *handler) startCallProc(fn func(*callProc)) {
 	h.callWG.Add(1)
-	gopool.Submit(func() {
+	go func() {
 		ctx, cancel := context.WithCancel(h.rootCtx)
 		defer h.callWG.Done()
 		defer cancel()
 		fn(&callProc{ctx: ctx})
```

Comment:
The context never expires, which should be fixed on the near future.

Reply:
upstream PR https://github.com/ethereum/go-ethereum/pull/25457 better handle RPC timeout, wait BSC merge v1.12.2 ?

Reply:
Agree.
But we could merge it on the other PR.